ci.jsonnet

```diff
@@ -563,7 +563,7 @@ local composition_environment = utils.add_inclusion_tracking(part_definitions, "
       local linux_gate = $.platform.linux + $.cap.gate + $.jdk.labsjdk8 + $.use.common + { timelimit: "01:00:00" },
 
       "ruby-lint": linux_gate + $.run.lint + { timelimit: "30:00" },  # timilimit override
-      "ruby-test-tck": linux_gate + $.use.build + { run+: [["mx", "rubytck"]] },
+      "ruby-test-tck": linux_gate + $.use.build + { run+: [["mx", "rubytck"], ["mx", "tck"]] },
```

Comment:
I think `mx tck` without `-Dtck.language=ruby` also runs it for other languages like SL, which might not be necessary or useful, not sure.

Reply:
That's the point isn't it? Run it with all available languages?

Reply:
Yeah, if it works it's probably better. cc @chumer